lib/py/src/compat.py

```diff
@@ -17,30 +17,13 @@
 # under the License.
 #
 
-import sys
+from io import BytesIO as BufferIO  # noqa
```

Comment:
why is this import needed? neither `BytesIO` nor `BufferIO` seems to be used in this file.

also, why the rename here?

Reply:
The two compat.py should go.

I planned to do this in the next PR

Reply:
ok looks like this is from old code